src/main/java/life/qbic/portal/portlet/OMEROClientPortlet.java

```diff
@@ -64,7 +68,8 @@
 import org.apache.logging.log4j.Logger;
 
 /**
- * Entry point for portlet omero-client-portlet. This class derives from {@link QBiCPortletUI}, which is found in the {@code portal-utils-lib} library.
+ * Entry point for portlet omero-client-portlet. This class derives from {@link QBiCPortletUI},
+ * which is found in the {@code portal-utils-lib} library.
```

Comment:
See other comments on documentation style

Reply:
I came, I saw and i Conquered the new documentation style as suggested 💪